api/src/db/migrations/v150_extract_business_data.ts

```diff
@@ -0,0 +1,878 @@
+import { v149Business, v149UserData } from "@db/migrations/v149_add_environment_data";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v149_to_v150 = (v149Data: v149UserData): v150UserData => {
+  return {
+    ...v149Data,
+    businesses: Object.fromEntries(
+      Object.values(v149Data.businesses)
+        .map((business: v149Business) => migrate_v149Business_to_v150Business(business))
+        .map((currBusiness: v150Business) => [currBusiness.id, currBusiness])
```

Comment:
You're right @jphechter ; this line has been in the migrations for a while, but the way I understand it is that it is reconstructing the K-V pairs ([id, business]) for Object.fromEntries. Since the id field isn’t being modified in the migration function, this line is redundant.
I’ve removed it in this migration to simplify the code while maintaining the same functionality. Thanks for pointing it out!